src/Frontend/Dfe.Complete/Pages/Projects/TaskList/Tasks/HandoverWithDeliveryOfficerTask/HandoverWithDeliveryOfficerTask.cshtml.cs

```diff
@@ -20,9 +23,36 @@ public class HandoverWithDeliveryOfficerTaskModel(ISender sender, IAuthorization
     [BindProperty(Name = "attend-handover-meeting")]
     public bool? AttendHandoverMeeting { get; set; }
 
+    [BindProperty]
+    public Guid? TasksDataId { get; set; }
+    [BindProperty]
+    public ProjectType? ProjectType { get; set; }
+
     public override async Task<IActionResult> OnGetAsync()
     {
-        await base.OnGetAsync(); 
+        await base.OnGetAsync();
+        ProjectType = Project.Type;
+        TasksDataId = Project.TasksDataId?.Value;
+        if (Project.Type == Domain.Enums.ProjectType.Transfer)
```

Comment:
Looks like we have a `using` in for the Domain.Enums already, can this simplify?
```suggestion
        if (Project.Type == ProjectType.Transfer)
```